tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -960,7 +962,88 @@ public void DateArchive_SkipPeriod(DateTimeKind timeKind, FileArchivePeriod arch
                     Directory.Delete(tempPath, true);
             }
         }
+        public static IEnumerable<object[]> DateArchive_AllLoggersTransferToCurrentLogFile_TestParameters    {
```

Comment:
PS: I was thinking to introduce https://github.com/AArnott/Xunit.Combinatorial in the future :)